lib/scopes.js

```diff
@@ -0,0 +1,29 @@
+var SCOPES = {
+  user: [
```

Comment:
:+1: This is a really easy way to see what scopes get you what data.

Is there any additional data we should add here? https://api.github.com/users/jbuck as inspiration